**Задача 1.**

*Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800]. Найдите ее среднее значение и дисперсию.*

Используем формулы для случайной непрерывной величины $X$, равномерно распределённой на отрезке $[a;b]$:  
$\displaystyle\text M(X)=\frac{a+b}{2}\\\displaystyle\text {Дисперсия: }D(X)=\frac{(b-a)^2}{12}$
:  

In [1]:
МА = (200+800)/2
DA = ((800-200)**2)/12
print(МА,DA)

500.0 30000.0


**Ответ:**

Среднее значение: М(А) =  500.00 

Дисперсия: D(A) =  30000.00

---  


**Задача 2.**

*О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2. Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5? Если да, найдите ее.*

из формулы:  
$\displaystyle D(B)=\frac{(b-a)^2}{12}$  
мы имеем значения D(B)=0.2 и a=0.5

Найдем правую границу b величины B из уравнения  
$\displaystyle \frac{(b-0.5)^2}{12}=0.2\\b-0.5=\sqrt{2.4}\\b=0.5+\sqrt{2.4}$  

потом найдем среднее значение по формуле:  
$\displaystyle M(B)=\frac{a+b}{2}$

In [2]:
b=0.5+2.4**(1/2)
MB =(b+0.5)/2
print(b, MB)

2.049193338482967 1.2745966692414834


**Ответ:**

Левая граница распределения: $b=2.049$  
Среднее значение: $M(B)=1.275$  

---

**Задача 3.**

*Непрерывная случайная величина X распределена нормально и задана плотностью распределения $\displaystyle f(x)=\frac{1}{4\cdot\sqrt{2\pi}}\cdot e^{-\frac{(x+2)^2}{32}}$, Найдите:*  

*а) M(X)*  
*б) D(X)*  
*в) std(X)(среднее квадратичное отклонение)*

Так как общая формула плотности нормального распределения выглядит как:  
$\displaystyle f(x)=\frac{1}{\sigma\sqrt{2\pi}}\cdot e^{-\frac{(x-a)^2}{2\sigma^2}}$  

то из нее применительно к заданной получаем:

$M(X)=a=-2\\D(X)=\sigma^2=16\\std(X)=\sigma = 4$

**Ответ:**

$M(X)=-2\\D(X)=16\\std(X)=4$  

---

**Задача 4.**

*Рост взрослого населения города X имеет нормальное распределение. Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см. Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:*  
*а) больше 182 см*  
*б) больше 190 см*  
*в) от 166 см до 190 см*  
*г) от 166 см до 182 см*  
*д) от 158 см до 190 см*  
*е) не выше 150 см или не ниже 190 см*  
*ё) не выше 150 см или не ниже 198 см*  
*ж) ниже 166 см.*  

проведем рассчеты через модуль  stats.scipy:

In [4]:
from scipy import stats

а) вероятность роста больше 182 см 

In [5]:
1 - stats.norm.cdf(182, loc=174, scale=8)

0.15865525393145707

$P(X\ge182)=0.1587$

б) вероятность роста больше 190 см 

In [6]:
1 - stats.norm.cdf(190, loc=174, scale=8)

0.02275013194817921

$P(X\ge190)=0.2275$

в) вероятность роста от 166 см до 190 см

In [7]:
(stats.norm.cdf(190, loc=174, scale=8))-(stats.norm.cdf(166, loc=174, scale=8))

0.8185946141203637

$P(166\le X\le 190)=0.8186$  

г) вероятность роста от 166 см до 182 см

In [8]:
(stats.norm.cdf(182, loc=174, scale=8))-(stats.norm.cdf(166, loc=174, scale=8))

0.6826894921370859

$P(166\le X\le 182)=0.6827$  

д) вероятность роста от 158 см до 190 см

In [9]:
(stats.norm.cdf(190, loc=174, scale=8))-(stats.norm.cdf(158, loc=174, scale=8))

0.9544997361036416

$P(158\le X\le 190)=0.9545$  

е) вероятность роста не выше 150 см или не ниже 190 см

In [10]:
(stats.norm.cdf(150, loc=174, scale=8))+(1 - stats.norm.cdf(190, loc=174, scale=8))

0.0241000299798093

$P(X\le150\cup X\ge 190)=0.0241$  

ё) вероятность роста не выше 150 см или не ниже 198 см

In [11]:
(stats.norm.cdf(150, loc=174, scale=8))+(1 - stats.norm.cdf(198, loc=174, scale=8))

0.0026997960632601965

$P(X\le150\cup X\ge 198)=0.0027$  

ж) вероятность роста ниже 166 см:

In [12]:
(stats.norm.cdf(166, loc=174, scale=8))

0.15865525393145707

$P(X\le166)=0.1587$  

---

**Ответ:**

а) вероятность роста больше 182 см = 0.1587;  
б) вероятность роста больше 190 см = 0.02275;  
в) вероятность роста от 166 см до 190 см = 0.8186;  
г) вероятность роста от 166 см до 182 см = 0.6827;  
д) вероятность роста от 158 см до 190 см = 0.9545;  
е) вероятность роста не выше 150 см или не ниже 190 см = 0.0241;  
ё) вероятность роста не выше 150 см или не ниже 198 см = 0.0027;  
ж) вероятность роста ниже 166 см = 0.1587;  

---

**Задача 5.**

*На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?*  

Воспользуемся формулой расчета $Z$-оценки:  
$\displaystyle Z=\frac{X-\bar x}{\sigma}=\frac{190-178}{\sqrt{25}}=\frac{12}{5}$    

In [13]:
Z=(190-178)/25**(1/2)
Z

2.4

**Ответ:**

на $2.4$ сигмы  

---